dydx3/modules/private.py

```diff
@@ -1012,11 +1012,14 @@ def create_fast_withdrawal(
             )
             signature = transfer_to_sign.sign(self.stark_private_key)
 
+        if type(slippage_tolerance) is float:
+            slippage_tolerance = str(slippage_tolerance)
+
```

Comment:
I think we shouldn't allow for float, its antithetical to the formatting of all our APIs

Reply:
updated!

Reply:
updated!